Build-Core.cmd

```diff
@@ -33,31 +38,31 @@ call %_dotnet% restore "%CMDHOME%\Build\Tools.csproj" --packages %TOOLS_PACKAGES
 
 SET CURRENT_CONFIGURATION=Debug
 
-call %_dotnet% restore /p:Configuration=%CURRENT_CONFIGURATION% "%SOLUTION%"
+call %_dotnet% restore /bl:%LOGFILENAME_PREFIX%-%CURRENT_CONFIGURATION%-Restore.binlog;ProjectImports=Embed /p:Configuration=%CURRENT_CONFIGURATION% "%SOLUTION%"
```

Comment:
`;ProjectImports=Embed` is not needed as its the default (unless it's on purpose for discoverability)